connexta/eve
# Bugfix: resolve left out promise during Unmounting

#### What does this PR do?
resolve left out promise during Unmounting which causes memory leak issue
- Applied trashable library to achieve it.

#### Who is reviewing it?

@vinamartin @Kjames5269 @bankent1 @alchucam @mcalcote @haydenudelson

#### How should this be tested?
Unmount the component during fetch call, and see if there's any warning appears.
In other words, enter and exit /tv quickly so that unmount happens during fetch call.

#### Screenshots

![image](https://user-images.githubusercontent.com/40583917/61496179-41fc5d00-a970-11e9-9efd-6cd6fb2aad95.png)
Before the bugfix, you will see this warnings but now the warning is gone!

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
